src/components/Dao/ProposalPluginCard.tsx

```diff
@@ -54,7 +54,7 @@ const ProposalPluginCard = (props: IProps) => {
           </TrainingTooltip> : headerHtml
         }
         <div>
-          <b>{pluginState.numberOfBoostedProposals}</b> <span>Boosted</span> <b>{pluginState.numberOfPreBoostedProposals}</b> <span>Pending Boosting</span> <b>{pluginState.numberOfQueuedProposals}</b> <span>Regular</span>
+          <b>{pluginState.numberOfBoostedProposals}</b> <span>Boosted   </span> <b>{pluginState.numberOfPreBoostedProposals}</b> <span>Pending Boosting   </span> <b>{pluginState.numberOfQueuedProposals}</b> <span>Regular</span>
```

Comment:
good opportunity to replace the `<b>` with css

Reply:
But still it'll be necessary to wrap it `<div>` or something since only the number is bold.

Reply:
of course